src/main/java/com/hazelcast/kubernetes/KubernetesClient.java

```diff
@@ -296,15 +299,20 @@ private static String extractZone(JsonObject nodeJson) {
                     // Load Balancer public IP cannot be found, try using NodePort.
                     Integer nodePort = extractNodePort(serviceJson);
                     String node = nodes.get(privateAddress);
-                    String nodePublicIp;
+                    String nodePublicAddress;
                     if (cachedNodePublicIps.containsKey(node)) {
-                        nodePublicIp = cachedNodePublicIps.get(node);
+                        nodePublicAddress = cachedNodePublicIps.get(node);
                     } else {
-                        String nodeUrl = String.format("%s/api/v1/nodes/%s", kubernetesMaster, node);
-                        nodePublicIp = extractNodePublicIp(callGet(nodeUrl));
-                        cachedNodePublicIps.put(node, nodePublicIp);
+                        if (useNodeNameAsExternalAddress) {
```

Comment:
Minor: consider extracting these 7 lines to a separate method.

Reply:
Annoying fact: when I extract that into a method, CheckStyle complains about `Total number of methods is 31 (max allowed is 30)`.

Any preferences about how to handle this?

Reply:
lol ;) Add `@SuppressWarnings` for now, we'll have to split this class into multiple classes, but let's not do it in this PR.

Reply:
Method extracted and `@SuppressWarnings("checkstyle:methodcount")` added.